src/assets/styles/modules/blog-post.scss

```diff
@@ -0,0 +1,15 @@
+/* ================ Blog post ================ */
+.blog-post {
+  display: grid;
```

Comment:
I feel like this approach might break if we increase the post count above 3. Maybe flex-box would be more accommodating?

Reply:
Yes, I use flexbox now. Since I don't have much experience with CSS grids, I wanted to give it a try and see how it works.